enhancements/windows-containers/cluster-logging-windows-worker-nodes.md

```diff
@@ -36,116 +35,87 @@ and pods running in an OpenShift cluster
+ * Configure td-agent to forward logs to the elasticsearch data storage in a format
+   understandable to elasticsearch. 
+ * Kubelet on Windows nodes logs to a file. So, we also need to configure 
+   fluentd's record transformer to ensure that logs collected from the kubelet log
+   file are in required format.
+          
 
 ### Justification
 
-Following are the reasons for having this ansible playbook approach instead of 
-an operator driven approach:
-
-* When we move to an operator based model, we would like to use the cluster 
-  logging operator for managing the logging on Windows nodes instead of
-  creating our own operator
-* Cluster logging operator will undergo changes related to the log collection 
-  API in the 4.3 timeframe and that uncertainity makes it difficult to introduce
-  Windows specific changes
-
-This design allows us to be future proof where in the ansible playbook can be 
-converted to an operator.
+* The reason we want to use the .msi installer package is to have the ability
+  to run Fluentd as a Windows service. This will allow Windows to manage the 
+  process and ensure it is always running for log collection.
+  
  
 ### Risks and Mitigations
 
-The main risk with this proposal are the following dependencies on Microsoft: 
-
-* To provide us with fluentd binary
-* Container logging works on Windows nodes
-* Container runtime writes container logs in a format understandable by fluentd
-  plugin
-
-Mitigations:
-
-* If fluentd binary is not shipped by Microsoft, we plan to install fluentd 
-  from upstream which will result in Red Hat being responsible for security and 
-  other fixes. 
-* If Microsoft doesn't provide container logging framework on Windows nodes, we
-  would just support node logging in 4.3 timeframe
-
-## Design Details
+The risks involved with this proposal are:
+* Dependency on the Microsoft provided td-agent .msi installer for providing a logging solution.
 
 ### Test Plan
 
-We plan to add e2e tests to ensure 
+We plan to add e2e tests to ensure:
 
-* Fluentd service is running on Windows node
-* Windows nodes are forwarding logs properly to elasticsearch
+* Fluentd is running as a Windows service on Windows nodes.
+* Windows nodes are forwarding logs properly to elasticsearch.
 
 ### Graduation Criteria
 
-This enhancement will start as GA
+This enhancement will start as GA.
 
 ### Upgrade / Downgrade Strategy
 
-We will support upgrades/downgrade of fluentd by publishing a new release of 
-Windows Node logging playbook. An older release of the playbook can be used to 
-downgrade.
-
+We will support upgrade / downgrade to the Fluentd component, 
+compatible with the release versions of WMCO. 
 
 ## Implementation History
 
-v1: Initial Proposal
+We had initially implemented a [ansible playbook](https://github.com/openshift/windows-machine-config-bootstrapper/tree/0cfab036cefbe4d658b160d9ae80289ce1e1249f/tools/ansible/tasks/logging) approach to install Fluentd
```

Comment:
This was never used or tested, I dont think this should be mentioned

Reply:
You can mention the enhancement but to say it was `implemented` isnt fully true. Saying `explored` would be more accurate. Although we really didnt spend much time on it at all